committee-steering/governance/sig-governance.md

```diff
@@ -117,13 +116,21 @@ curation from other SIG participants
 
 ### Tech Lead
 
-- *Optional Role*: SIG Technical Leads
-  - Establish new subprojects
-  - Decommission existing subprojects
-  - Resolve X-Subproject technical issues and decisions
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
-  - Role description in [technical-lead.md]
+- Number: 2+
+- Membership tracked in [sigs.yaml]
+- *MUST* Approve & facilitate the creation of new subprojects
+- *MUST* Approve & facilitate decommissioning of existing subprojects
+- *MUST* Resolve X-Subproject and X-SIG technical issues and decisions
+  or delegate to another Lead as needed
+- *MUST* in conjunction with the Chairs identify, track, and maintain the
+  metadata of the SIGs enhancement proposals for the current release and serve
+  as point of contact for the release team, but *MAY* delegate to another
+  Lead to fulfill these responsibilities
+- *MUST* Review & Approve SIG Enhancement Proposals, but *MAY* delegate to
+  another Lead to fulfill these responsibilities for individual proposals
```

Comment:
```suggestion
  another Lead or contributor to fulfill these responsibilities for individual proposals
```

maybe approver or area owner

Reply:
updated all language to be "other contributors".